### PR TITLE
#2319 Added functionality to show a download link to Keptn Bridge CLI

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
This PR introduces a download-link for Keptn CLI.

In addition, I added the `rel="noopener noreferrer"` in the app-header component.

Other changes:
* Updated `deploy.yaml` with the download link
* Updated bridge installer manifest to also contain the download link to the github releases page based on appVersion
* Moved the `index` api endpoint of `bridge/api/` to `bridge/api/bridgeInfo` (this goes along with what it is called in the Angular UI)
* Whatever has been done to the package.lock.json is beyond my knowledge

image

### CODE REVIEW

File: installer/manifests/keptn/charts/control-plane/templates/core.yaml [hunk 1]
```diff
@@ -168,6 +168,8 @@ spec:
                 secretKeyRef:
                   name: keptn-api-token
                   key: keptn-api-token
+            - name: CLI_DOWNLOAD_LINK
+              value: {{ .Values.bridge.cliDownloadLink | default (print "https://github.com/keptn/keptn/releases/tag/" .Chart.AppVersion) }}
```
Comment (christian-kreuzberger-dtx) [inline installer/manifests/keptn/charts/control-plane/templates/core.yaml:172]: I have tested this with @agrimmer

It basically uses the download link if it is available, else it builds a download-link that points to the github releases page, e.g., https://github.com/keptn/keptn/releases/tag/0.7.1

File: bridge/client/app/app-header/app-header.component.html [hunk 2]
```diff
@@ -19,6 +19,12 @@
 </dt-top-bar-navigation>
 <dt-context-dialog #dialog aria-label="User menu" ariaLabelClose="Close user menu">
   <h3 class="m-0">Get started</h3>
+  <div class="mt-1" fxLayout="row" fxLayoutAlign="flex-start center" *ngIf="keptnInfo?.bridgeInfo?.cliDownloadLink">
+    <div fxFlex>
+      <p class="m-0 mr-2 nobreak">
+        <a [attr.href]="keptnInfo.bridgeInfo.cliDownloadLink" rel="noopener noreferrer" target="_blank">Download keptn CLI</a></p>
```
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.html:25]: Can you please change it to a capital K --> _Download Keptn CLI_ 

FYI: We stopped differentiating between "keptn CLI" and "Keptn". In the docs it is always Keptn CLI (when not used as command blocks.)

**Comment by johannes-b** [comment]
I tested the new feature and it works fine!

File: bridge/client/app/app-header/app-header.component.html (hunk 2 above)
Comment (christian-kreuzberger-dtx) [reply bridge/client/app/app-header/app-header.component.html]: Done

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**